src/gap.c

```diff
@@ -897,6 +897,32 @@ static Obj FuncTotalMemoryAllocated(Obj self)
     return ObjInt_UInt8(SizeAllBags);
 }
 
+/****************************************************************************
+**
+*F  SyDisableMemoryLimits() . . . . .  Disable limits on workspace size
+**
+**  By default GASMAN limits the size of the workspace using SyStorMax and
+**  SyStorKill. This disables those checks.
+*/
+static Obj FuncDisableMemoryLimits(Obj self)
+{
+#ifdef SYS_IS_64_BIT
+    int64_t max_mem = INT64_MAX;
+#else
+    int32_t max_mem = INT32_MAX;
+#endif
+
+#if defined(USE_GASMAN) || defined(USE_BOEHM_GC)
+    SyStorKill = max_mem;
+#endif
+
+#if defined(USE_GASMAN)
+    SyStorMax = max_mem;
+#endif
+
```

Comment:
```suggestion
#if defined(USE_GASMAN) || defined(USE_BOEHM_GC)

#ifdef SYS_IS_64_BIT
    int64_t max_mem = INT64_MAX;
#else
    int32_t max_mem = INT32_MAX;
#endif

    SyStorKill = max_mem;

#if defined(USE_GASMAN)
    SyStorMax = max_mem;
#endif

#endif // defined(USE_GASMAN) || defined(USE_BOEHM_GC)

```
Otherwise we'll get compiler warnings about `max_mem` being unused when building for Julia (I think)